packages/js/api-core-tests/bin/wc-api-tests.sh

```diff
@@ -35,7 +35,7 @@ report() {
 	REPORT_EXIT_CODE=$?
 
 	# Suggest opening the report
-	if [ $REPORT_EXIT_CODE -eq 0 ]; then
+	if [[ $REPORT_EXIT_CODE -eq 0 && $GITHUB_ACTIONS != "true" ]]; then
```

Comment:
The message being printed here only makes sense when the user's running this script on their local machine. Adding `$GITHUB_ACTIONS != "true"` as a condition will not print this message when this script is being run on CI.